warrior/Framework/Utils/xml_Utils.py

```diff
@@ -986,7 +986,9 @@ def compare_xml_using_xpath(response, list_of_xpath, list_of_expected_api_respon
         for index, xpath_pattern in enumerate(list_of_xpath):
             xpath = xpath_pattern.strip("xpath=")
             value = getValuebyTagFromStringWithXpath(response, xpath, None)
-            if value != list_of_expected_api_responses[index]:
+            # Check if the expected value/pattern is in actual response
+            match = re.search(list_of_expected_api_responses[index], value)
+            if not match:
```

Comment:
Hi @venkat1077,

Can we do both, the equality check and regex searching - so as to avoid a backward incompatibility?

Thanks,
Sanika

Reply:
Sure, Thanks @sanikakulkarni 
Made the required changes. Now, the equality check will happen first and if it fails, regex check will happen.